pcsx2/Gif_Unit.h

```diff
@@ -258,9 +278,10 @@ struct Gif_Path {
 
 	// If completed a GS packet (with EOP) then returned GS_Packet.done = 1
 	// MTVU: This function only should be called called on EE thread
-	GS_Packet ExecuteGSPacket() {
+	GS_Packet ExecuteGSPacket(bool &done) {
```

Comment:
Actually, it might be nice to have the parameter as a pointer instead. That way we could easily understand that the variable's original value might be modified when we take a look at the function call. ``path.ExecuteGSPacket(&done)`` **instead of** ``path.ExecuteGSPacket(done)``. Any opinion?